plugins/libplugin.c

```diff
@@ -533,8 +533,7 @@ handle_getmanifest(struct command *getmanifest_cmd)
 		json_add_string(params, NULL, p->hook_subs[i].name);
 	json_array_end(params);
 
-	json_add_string(params, "dynamic",
-			p->restartability == PLUGIN_RESTARTABLE ? "true" : "false");
+	json_add_bool(params, "dynamic", p->restartability);
```

Comment:
`restartability` is of type `enum plugin_restartability`, not `bool`.  Suggest doing:

    switch (p->restartability) {
    case PLUGIN_STATIC:
        json_add_bool(params, "dynamic", false);
        break;
    case PLUGIN_RESTARTABLE:
        json_add_bool(params, "dynamic", true);
        break;
    }

Reply:
oops thanks !

Reply:
I'll go with adding `p->restartability == PLUGIN_RESTARTABLE`

Reply:
`switch` means we can add more `enum`s later and we will get warned about this code, but I suppose nobody will add more `enum` entries to `enum plugin_restartability`.